prac_08/silver_service_taxi.py

```diff
@@ -0,0 +1,17 @@
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    price_per_km = 1.23
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
+        super().__init__(name, fuel)
+        self.fanciness = fanciness
+        self.current_fare_distance = 0
+        self.initial_fare = SilverServiceTaxi.price_per_km * self.fanciness
```

Comment:
What's this? 
SST should reuse the existing price_per_km. It should not add a new attribute other than fanciness and flagfall.